### PR TITLE
Set Soft Power Off to be the default shutdown mode

### PR DESCRIPTION
**Pull request by Xenwar**
Co-authored-by: Jaakko Kuuskoski <jaakko.kuuskoski@est.tech>
Co-authored-by: maelk <mael.kimmerlin@est.tech>

### CODE REVIEW

**Comment by nordixinfra** [comment]
Can one of the admins verify this patch?

File: pkg/controller/baremetalhost/baremetalhost_controller.go [hunk 1]
```diff
@@ -827,6 +840,41 @@ func (r *ReconcileBareMetalHost) saveStatus(host *metal3v1alpha1.BareMetalHost)
...
+}
+
+// Remove soft power off time stamp
+func (r *ReconcileBareMetalHost) unMarkLastSoftPowerOff(host *metal3v1alpha1.BareMetalHost) error {
+	host.Status.LastSoftPowerOff = nil
+	return r.client.Status().Update(context.TODO(), host)
+}
+
+// Check if Soft Power Off takes longer than timeout treshold, for any reason.
+func (r *ReconcileBareMetalHost) softPowerOffTimeoutNotReached(host *metal3v1alpha1.BareMetalHost) bool {
```
Comment (dhellmann) [inline pkg/controller/baremetalhost/baremetalhost_controller.go]: Won't ironic manage the timeout for us?

Comment (Xenwar) [reply pkg/controller/baremetalhost/baremetalhost_controller.go]: It does. But, we need to cover the case where ironic does not move to hard power off, for any reason. 
Basically, trying to enable the control to make the final say(regardless of ironic)

**Comment by Xenwar** [comment]
> How does this work relate to #294?

Em, there seems to be a duplicate work. I think we can go with #294.

**Comment by hardys** [comment]
> Em, there seems to be a duplicate work. I think we can go with #294.

Closing per this comment

**Pull request closed without merging**